daemons/controld/controld_based.c

```diff
@@ -31,7 +31,7 @@ do_cib_updated(const char *event, xmlNode * msg)
 void
 do_cib_replaced(const char *event, xmlNode * msg)
 {
-    int change_section = cib_change_section_nodes | cib_change_section_status;
+    long long change_section = cib_change_section_none;
```

Comment:
This defaults to nodes|status because F_CIB_CHANGE_SECTION is a new capability that older pacemaker-based's won't provide, and we need to preserve the old behavior for them.